eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -53,9 +55,21 @@ object Graph {
     require(ageFactor >= 0.0, "ratio-channel-age must be nonnegative")
     require(capacityFactor >= 0.0, "ratio-channel-capacity must be nonnegative")
   }
+
+  /**
+   * We use heuristics to calculate the weight of an edge.
+   * The fee for a failed attempt and the fee per hop are never actually spent, they are used to incentivize shorter
+   * paths or path with higher success probability.
+   *
+   * @param lockedFundsRisk cost of having funds locked in htlc in msat per msat per block
+   * @param failureCost     fee for a failed attempt
+   * @param hopCost         virtual fee per hop (how much we're willing to pay to make the route one hop shorter)
+   */
+  case class HeuristicsConstants(lockedFundsRisk: Double, failureCost: RelayFees, hopCost: RelayFees)
```

Comment:
I don't think `HeuristicsConstants` is a good name for this, especially if we introduce a `sealed trait` as described in my other comment. This class should reflect the fact that the heuristic it uses is the cost of failure (`FundsLockedParams` or `FundsLockedHeuristics` maybe?).